cime_config/config_component.xml

```diff
@@ -303,11 +304,12 @@
 
   <entry id="CAM_LINKED_LIBS">
     <type>char</type>
-    <valid_values></valid_values>
+    <valid_values>-lmpas</valid_values>
     <default_value>-lmusica -ljsonfortran</default_value>
     <values match="last">
-      <!--Turn off for physics testbed -->
-      <value compset="_CAM%PHYSTEST"> </value>
+      <!-- Turn off for physics testbed -->
+      <value compset="_CAM%PHYSTEST"></value>
+      <value grid="a%mpasa[0-9]+">-lmpas</value>
```

Comment:
This is fine as long as there are no immediate plans to run MPAS in CAM-SIMA with chemistry.  Otherwise we'll need to modify the `<value match="last">` line above to be `<value match="last" modifier="additive">`.

Reply:
Of course. I made the suggested change in https://github.com/ESCOMP/CAM-SIMA/commit/3a9b8ad2c0ceecf740a60c40dfade1c78eff1f74.